core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -607,6 +608,8 @@
     public static final String WORKER_DATA_HOSTNAME = "alluxio.worker.data.hostname";
     public static final String WORKER_DATA_PORT = "alluxio.worker.data.port";
     public static final String WORKER_DATA_SERVER_CLASS = "alluxio.worker.data.server.class";
+    public static final String WORKER_DATA_SERVER_DOMAIN_SOCKET_ADDRESS =
+        "alluxio.worker.data.server.domain.socket.address";
```

Comment:
I do not see any reason to have more than 1. AFAIK, this address plays similar role as port.